### PR TITLE
Use `extract_plain_text` to render plain text in `/serials`

### PR DESCRIPTION
**Pull request by okkez**
connect to https://github.com/bm-sms/daimon-news-misc/issues/147

### CODE REVIEW

File: test/integration/serial_test.rb [hunk 1]
```diff
@@ -56,5 +68,23 @@ class SerialTest < ActionDispatch::IntegrationTest
...
+      # h1
+
+      contents
+      DESC
+      visit "/serials"
+
+      click_on serial.title
+
+      within ".serial-content__description" do
+        h1 = find "h1"
```
Comment (tricknotes) [inline test/integration/serial_test.rb]: `find("h1")` ?

**Comment by cafedomancer** [comment]
Looks like it's working for me. Any thoughts?

**Comment by tricknotes** [comment]
It works 👍 
I think this PR is merge ready when https://github.com/bm-sms/daimon-news/pull/558#discussion_r68879511 is fixed.

File: test/integration/serial_test.rb (hunk 1 above)
Comment (cafedomancer) [inline test/integration/serial_test.rb]: The above comment insists that we should use the `find` method or other test helpers with parentheses? More descriptions help me to write better codes for our team. 😆

Comment (okkez) [inline test/integration/serial_test.rb]: `find` is Capybara DSL, I think.
So I don't use `find` method without parentheses.

According to our coding standard. See esa, please.

File: test/integration/serial_test.rb [hunk 2]
```diff
@@ -37,6 +37,18 @@ class SerialTest < ActionDispatch::IntegrationTest
       number_of_posts = find_all ".number-of-posts"
       assert_equal(["記事数：1", "記事数：1", "記事数：1"], number_of_posts.map(&:text))
     end
+
+    test "markdown" do
+      create(:serial, :with_posts, site: @site, description: <<~DESC)
+      # h1
+
+      contents
+      DESC
```
Comment (cafedomancer) [inline test/integration/serial_test.rb:46]: May need to indent heredocs?

**Comment by tricknotes** [comment]
🚀

**Pull request merged**